Script/PRONTO.py

```diff
@@ -231,7 +232,123 @@ def clear_blank_line(file_in,file_out):
 	os.remove(file_in)
 
 
-def get_patient_info_from_MTF(ipd_material_file,ipd_no,DNA_sampleID,RNA_sampleID):
+def get_patient_info_from_MTF_2023(ipd_material_file,ipd_no,DNA_sampleID,RNA_sampleID):
+	import xlrd
+	global ipd_birth_year
+	global ipd_gender
+	global ipd_consent
+	global DNA_material_id
+	global RNA_material_id
+	global ipd_collection_year
+	global requisition_hospital
+	global extraction_hospital
+	global batch_nr
+	global tumor_content_nr
+	global inclusion_site
+	open_exl_material = xlrd.open_workbook(ipd_material_file)
+	sheet_material = open_exl_material.sheet_by_index(0)
+	nrows_material = sheet_material.nrows
+	ncols_material = sheet_material.ncols
+	col_ipd = "InPreD ID"
```

Comment:
I understand that these are all column names? Maybe they can be collected in an object?